Jenkinsfile

```diff
@@ -60,14 +60,18 @@ pipeline {
             }
         }
         stage('Integration tests') {
-            agent {
-                docker {
-                    image 'maven:3.6.3-jdk-11'
-                    label 'docker'
-                }
-            }
+            agent { label 'docker'}
             steps {
-                sh 'bin/devcontrol.sh backend integration-tests'
+                script {
+                    docker.image('docker:dind').withRun('--privileged -v "$WORKSPACE":"$WORKSPACE" --workdir "$WORKSPACE"') { c ->
```

Comment:
Wouldn't it be better if we create a docker image with all the dependencies we need and upload it to docker hub instead or creating one from scratch every time we launch the pipeline?

Reply:
Yes, you're right. It's better to have a Dockerfile and build && push a docker image (maybe in a previous step).

I will change that shortly